ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg

```diff
@@ -53,11 +53,13 @@ interface Vlan83
 interface Vlan84
    description SVI Description
    ip address 10.10.84.1/24
-   ip virtual-router address 10.10.84.254
```

Comment:
This line is being removed with your changes. We should never accept things disappearing from molecule unless we intended so. In this case you should remember to update the data model for vlan84 as well.